opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -111,4 +122,15 @@
      * @return this Span instance, for chaining
      */
     Span setOperationName(String operationName);
+
+    /**
+     * @deprecated use {@link #log(Map)} like this
+     * {@code span.log(Map.of("event", "timeout", "millis", 1500))}
```

Comment:
this is close, but not the best example, as the doc is on a single-event method. I'd leave out the "millis"